src/bip32.c

```diff
@@ -735,6 +735,11 @@ int bip32_key_from_base58_alloc(const char *base58,
     return ret;
 }
 
+void bip32_key_strip_private_key(struct ext_key *hdkey)
+{
+    key_strip_private_key(hdkey);
```

Comment:
it should check for a null pointer and bail out.

Reply:
Done